src/main/scala/census/db/creator/DatabaseCreator.scala

```diff
@@ -0,0 +1,55 @@
+package census.db.creator
+import java.nio.file.Paths
+
+import akka.actor.{ActorSystem, Props}
+import akka.pattern.ask
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+import census.db.creator.config.Hardcoded
+import census.db.creator.database.DbMigrationHandler
+import census.db.creator.service.actors._
+import census.db.creator.service.fileDownloader.FileDownloadService
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+/*
+
+RUN POSTGIS DOCKER WITH FOLLOWING COMMAND:
+
+docker run --name postgis -e POSTGRES_PASSWORD=postgres1 -e POSTGRES_DB=census -p 5432:5432 -d mdillon/postgis
+
+ */
+
+private[creator] object DatabaseCreator extends App {
```

Comment:
App is generally not suggested btw - it's probably fine here, but also good practice to build your own main method